packages/mail/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@sendgrid/mail",
   "description": "Twilio SendGrid NodeJS mail service",
-  "version": "7.4.0",
+  "version": "7.4.1",
```

Comment:
Don't bump package version here. It will be bumped automatically during the next release. Other than that this PR looks good.

Reply:
reverted version bump for mail package